jobs/run_all_metrics.py

```diff
@@ -18,73 +20,49 @@ def run_table(combo):
 
     try:
         return summary_metrics_table(start_time, end_time, variable, truth, metric,
-                              time_grouping=time_grouping, grid=grid, region=region,
-                              force_overwrite=True, filepath_only=True,
-                              recompute=recompute, storage_backend=backend)
+                                     time_grouping=time_grouping, grid=grid, region=region,
+                                     force_overwrite=True, filepath_only=True,
+                                     recompute=recompute, storage_backend=backend)
     except KeyboardInterrupt as e:
-        raise(e)
+        raise (e)
     except NotImplementedError:
-        print(f"Metric {forecast} {lead} {grid} {variable} {metric} not implemented: {traceback.format_exc()}")
+        print(f"Metric {grid} {variable} {metric} {region} {time_grouping} not implemented: {traceback.format_exc()}")
         return "Not Impelemnted"
     except:  # noqa: E722
         print(f"Failed to run metric {grid} {variable} {metric} \
                 {region} {time_grouping}: {traceback.format_exc()}")
         return None
 
+
 def run_grouped(combo):
     """Run grouped metrics."""
     print(combo)
     metric, variable, grid, region, lead, forecast, time_grouping, truth = combo
 
     try:
         return grouped_metric(start_time, end_time, variable, lead, forecast, truth, metric,
-                       spatial=False, time_grouping=time_grouping, grid=grid, region=region,
-                       force_overwrite=True, filepath_only=True, recompute=recompute)
+                              spatial=False, time_grouping=time_grouping, grid=grid, region=region,
+                              force_overwrite=True, filepath_only=True, recompute=recompute)
     except KeyboardInterrupt as e:
-        raise(e)
+        raise (e)
     except NotImplementedError:
         print(f"Metric {forecast} {lead} {grid} {variable} {metric} not implemented: {traceback.format_exc()}")
         return "Not Impelemnted"
-    except: # noqa:E722
-        print(f"Failed to run global metric {forecast} {lead} {grid} {variable} {metric}: {traceback.format_exc()}")
+    except:  # noqa:E722
+        print(f"Failed to run grouped metric {forecast} {lead} {grid} {variable} {metric}: {traceback.format_exc()}")
         return None
 
-def run_global(combo):
-    """Run global metrics."""
-    metric, variable, grid, _, lead, forecast, _, truth = combo
-
-    if metric == 'rmse':
-        metric = 'mse'
-
-    try:
-        return global_metric(start_time, end_time, variable, lead, forecast, truth, metric, grid=grid,
```

Comment:
No sense of a global metric now - could update this to run global statistics, if we want to stick to the staged running paradigm?